src/Packetery/Module/Options/Provider.php

```diff
@@ -469,4 +472,41 @@ public function getCarrierLabelFormat(): array {
 
 		return $carrierLabelFormats;
 	}
+
+	/**
+	 * Force order status change on packet submit enabled.
+	 *
+	 * @return bool
+	 */
+	public function isOrderStatusAutoChangeEnabled(): bool {
+		$value = $this->get( 'order_status_auto_change' );
```

Comment:
Prosím , pojmenuj proměnou podle toho, co do ní ukládáš.  "value" je příliš obecné.

Reply:
V ostatních metodách je to také pojmenované jako $value. Buďme prosím konsistentní. Ta proměnná je v kontextu metody jedna jediná. Proto není potřeba mít jí pojmenovanou specificky.

Reply:
property $value bych nepoužíval nikde.  Ani tady , ani jinde.  Pokud to jinde máme, tak vytvoříme ticket pro nějakého juniora a přejmenujeme to i na dalších místech

Reply:
Prosím o konsistenci. Pokud se to má předělat, tak ať je to všude najednou. Navrhuji na to vytvořit ticket.

Reply:
Pracujeme s dev verzí , kde chvilkový nekonzistence nastat může.  Nové property v nový PR už prosím pojmenovat lépe. Včetně tohoto PR.  Na změnu property, které v dev větvi již máme jsem vytvořil ticket a změníme je co nejdříve to bude možné.